### PR TITLE
proofreading

### PR DESCRIPTION
**Pull request by iruzevic**
This is the text that was returned from the lectors. We should check if something is wrong here

### CODE REVIEW

File: website/docs/advanced/additional-libraries.md [hunk 1]
```diff
@@ -6,25 +6,25 @@ sidebar_label: Additional Libraries
...
+## Eightshift Libs
 
-This library is aimed at bringing the modern backend development tools to the [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate) or [Eightshift Boilerplate Plugin](https://github.com/infinum/eightshift-boilerplate-plugin), but you can use it on any WordPress project.
+This library is aimed at bringing the modern back-end development tools to the [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate) or [Eightshift Boilerplate Plugin](https://github.com/infinum/eightshift-boilerplate-plugin), but you can use it on any WordPress project.
 
 * [Documentation](/eightshift-docs/docs/eightshift-libs)
 * [Github](https://github.com/infinum/eightshift-libs)
 
-## Eightshift Frontend Libs 
+## Eightshift Front-End Libs
```
Comment (IvanGrginovInf) [inline website/docs/advanced/additional-libraries.md:25]: 🙅

File: website/docs/advanced/custom-eightshift-frontend-libs-usage.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: custom-eightshift-frontend-libs-usage
-title: Custom Eightshift Frontend Libs Usage
-sidebar_label: Custom Eightshift Frontend Libs Usage
+title: Custom Eightshift Front-end Libs Usage
+sidebar_label: Custom Eightshift Front-end Libs Usage
```
Comment (IvanGrginovInf) [inline website/docs/advanced/custom-eightshift-frontend-libs-usage.md:4]: **No** again 😄

**Review by IvanGrginovInf — CHANGES_REQUESTED** [review]
I'd definitely remove the `frontend libs` -> `front-end libs` changes. Other than that looks good. What do you think? It kinda changes are branding at very least and using `frontend` is pretty common among devs, no need to change it IMO.

**Review by dingo-d — APPROVED** [review]
Agree with @IvanGrginovInf about the name of the libs, otherwise looks ok 👍🏼

**Comment by iruzevic** [comment]
agreed will change it

**Comment by iruzevic** [comment]
fixed

**Pull request merged**